engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -26,20 +26,43 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
+ * A layout that allows for a single widget to be displayed among a list of stored widgets
+ */
 public class CardLayout extends CoreLayout<LayoutHint> {
+    /**
+     * The id of the currently displayed widget
+     */
```

Comment:
I'm not sure but I think if the variable you are documenting is private you should make this a standard comment rather than a javadoc. They idea behind javadoc is to browse them through a web-interface, with no code in sight. On the other hand, as a comment while browsing the source code it would be quite useful.